src/harness/tests/cbsd_sas.cfg

```diff
@@ -0,0 +1,10 @@
+[SasConfig]
```

Comment:
prefer not to put it under tests
prefer name it as sas.cfg since I think sas to sas test harness will share the same file.